eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -153,7 +154,7 @@ stages:
               - deployment: PublicDocsMS
                 displayName: Publish to Docs.MS
                 condition: and(succeeded(), ne(variables['Skip.PublishDocs'], 'true'))
-                environment: githubio
+                environment: ${{parameters.Environment}}
```

Comment:
Will We publish everything to nuget only? It used to be github

Reply:
This is just a DevOps environment and so we are trying to reduce our releases down to one environment as the multiple isn't necessary.